src/components/Settings/OverrideRule/OverrideRuleTiles.tsx

```
@@ -140,7 +140,7 @@ const OverrideRuleTiles = ({
   return (
     <>
       {rules.map((rule) => (
-        <li className="flex h-full flex-col rounded-lg bg-gray-800 text-left shadow ring-1 ring-gray-500">
+        <li className="flex h-full flex-col rounded-lg text-left shadow ring-1 bg-surface ring-ring-app/10">
```

Biome (2.4.6)

[error] Missing key property for this element in iterable.

(lint/correctness/useJsxKeyInIterable)